realis_e2e_test.go

```diff
@@ -1445,3 +1447,49 @@ func TestRealisClient_FitTasks(t *testing.T) {
 		}
 	}
 }
+
+func TestRealisClient_JobExists(t *testing.T) {
+	role := "vagrant"
+	env := "prod"
+	name := "test_job_exists"
+	// Create a good single job
+	job := realis.NewJob().
```

Comment:
can we add another call `exists, err := r.JobExists(job.JobKey())`  before creating the job?

Reply:
Done